src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -74,8 +68,8 @@ public static void tearDown() {
     @Test
     public void canCreateApiKeyForSelf() {
         assumeTrue(getBooleanEnvVar("RUN_E2E"));
-        HttpResponse<String> response = createApiKeyRequest(apiUser.id, apiUser.auth0UserId);
-        assertEquals(200, response.statusCode());
+        HttpResponse<String> response = createApiKeyRequest(apiUser.id, apiUser);
```

Comment:
I think this piece broke, it was previously containing the added API key, and with this change it is not.